build/teamcity/cockroach/ci/tests/sqllogic_hi_vmodule_nightly.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
```

Comment:
This should be in `build/teamcity/cockroach/nightlies`.

Reply:
Ooops, done.